contracts/casino/src/casino.cpp

```diff
@@ -163,4 +199,82 @@ uint32_t casino::get_profit_margin(uint64_t game_id) const {
     return platform::read::get_game(get_platform(), game_id).profit_margin;
 }
 
+void casino::set_bonus_admin(name new_admin) {
+    require_auth(get_owner());
+    check(is_account(new_admin), "new bonus admin account does not exist");
+    bstate.admin = new_admin;
+}
+
+void casino::deposit_bonus(asset quantity, const std::string& memo) {
```

Comment:
problem with pools separation not solved

Reply:
@algys removed the method
handling via transfer now